apps/electric_migrations/lib/electric_migrations/sqlite/introspect.ex

```diff
@@ -0,0 +1,127 @@
+defmodule ElectricMigrations.Sqlite.Introspect do
+  alias Exqlite.Sqlite3
+  alias ElectricMigrations.Ast
+
+  @spec open_in_memory! :: Sqlite3.db()
+  def open_in_memory!() do
```

Comment:
Because I'm doing a hard match against an ok-tuple and returning a pure conn. This is to contrast it against `Sqlite3.open` (without an exclamation mark) which doesn't raise but returns an ok-tuple